frontend/doc-site/032__Contribute and Publish Data.mdx

```diff
@@ -0,0 +1,56 @@
+# Contributing Data
+
+Cellxgene has a rapidly growing single cell data corpus because of generous contributions from researchers like you! If you feel like you have a single cell dataset that meets our requirments (see below), then feel free to reach out to our curation team at [cellxgene@chanzuckerberg.com](cellxgene@chanzuckerberg.com).
+
+## Publishing Process
+
+The general process for submission to the portal looks like this:
+- reach out to our curation team at [cellxgene@chanzuckerberg.com](cellxgene@chanzuckerberg.com) with a description of the dataset that you'd like to contribute. Generally speaking this would be derived from either human or mouse. We not allow submission of data sets from:
+  - organisms other than human or mouse
+  - cell lines
+  - drug screens 
+- recieve confirmation form the curation team will accept your dataset along with instructions on how to format your dataset for submission.
+- Prepare your dataset and email our curators for further assistance in the process
+- Publish!
+
+
+
+## Dataset Requirements
+
+We need the following collection metadata (i.e. details associate with your publication or study)
+
+- Collection information:
+  - Title
+  - Description
+  - Contact: name and email
+  - Publication/preprint DOI: can be added later
+  - URLs: any additional URLs for related data or resources, such as GEO or protocols.io - can be added later
+ 
+Datasets need the following information added to a single h5ad (AnnData) format file per cellxgene visualization:
+- **Dataset-level metadata in uns**:
+  - schema_version: '2.0.0'
+  - title: title of the individual dataset visualization
+  - X_normalization: method used to normalize the data stored in X and any scaling or logging; 'none' if data in X is raw 
+  - optional: batch_condition: list of obs fields that define that “batches” that a normalization or integration algorithm should be aware of
+- **Data in .X and raw.X**:
+  - raw counts are required
+  - normalized counts are strongly recommended
+  - raw counts should be in raw.X if normalized counts are in .X
+  - if there is no normalized matrix, raw counts should be in .X
+- **Cells metadata in obs (the most specific term available from the specified ontology)**:
```

Comment:
I think we should try to make this more clear or we'll get a bunch of 'scRNA-seq' data from 'adult'
**Cells metadata in ob - the values MUST be the most specific term available from the specified ontology**
(See schema for guidance and examples)